server/src/create-svg.es6

```diff
@@ -0,0 +1,89 @@
+const util = require('./util.es6');
+const jsdom = require('jsdom');
+const { JSDOM } = jsdom;
+const moment = require('moment');
+const fs = require('fs-extra');
+
+const createPermit = {};
```

Comment:
if this were made a class then you wouldn't have to pass the "frag" you could just have the permit fragment as property in the class, or are you not able to because of the async `fs`?

Reply:
not able to move "frag" likely because async `fs`